Apps/phokta/okta.json

```diff
@@ -15,6 +15,99 @@
             "type": "test",
             "verbose": ""
         },
+        {
+            "action": "send push notification",
+            "description": "Validate the user prompt with push notification",
+            "read_only": false,
+            "parameters": {
+                "email": {
+                    "required": true,
+                    "description": "Okta User id, email address, or username.",
+                    "data_type": "string",
+                    "default": "",
+                    "contains": [
+                        "okta user id"
+                    ],
+                    "primary": true,
+                    "order": 0
+                },
+                "factortype": {
+                    "required": true,
+                    "description": "Okta Factor Type.",
+                    "data_type": "string",
+                    "default": "push",
+                    "value_list": [
+                        "push",
+                        "sms",
+                        "token:software:totp"
+                    ],
+                    "order": 1
+                }
+            },
+            "versions": "EQ(*)",
+            "render": {
+                "type": "table"
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
```

Comment:
Add `column_name` and `column_order` to at least one `output` object so that there are UI results after running the action

Reply:
Resolved in 66dd58d  :)